pilot/pkg/model/config.go

```diff
@@ -72,6 +72,20 @@ func (key ConfigKey) String() string {
 	return key.Kind.String() + "/" + key.Namespace + "/" + key.Name
 }
 
+// HasConfigsOfKind returns true if configs has changes of type kind
+func HasConfigsOfKind(configs sets.Set[ConfigKey], kind kind.Kind) bool {
```

Comment:
This seems confusing with below `ConfigsOfKind`

Reply:
yeah it's just to avoid allocating a list really